### ISSUE TITLE
I18n conflicts with Activemodel I18n

### ISSUE THREAD

**Issue by be9** [codegram/date_validator#65]
ActiveModel contains own definition of `errors.messages.equal_to`, as seen in [here](https://github.com/rails/rails/blob/master/activemodel/lib/active_model/locale/en.yml#L31). date_validator redefines it [here](https://github.com/codegram/date_validator/blob/master/locales%2Fen.yml#L9) which causes breakage. E.g. I get following in my specs:

```
    I18n::MissingInterpolationArgument:
       missing interpolation argument :date in "must be equal to %{date}" ({:model=>"Site", :attribute=>"Monthly payment", :value=>1.0, :count=>0} given)
```

ActiveModel variant is `"must be equal to %{count}"` and it works if date_validator is not loaded.

**Comment by dolzenko** [comment]
Same issue here, I assume date_validator should use namespaced errors?

**Comment by thibaudgg** [comment]
:+1:

**Comment by eebs** [comment]
I have run into this problem as well. @dolzenko's fix worked for me as well, but I presume this would break date_validator's `equal_to` option message.

**Comment by mrcasals** [comment]
Closed by #77
